desktopexporter/internal/telemetry/sample.go

```diff
@@ -184,7 +184,8 @@ func fillCurrencyScope(scope pcommon.InstrumentationScope) {
 
 // requests scope data
 func fillRequestsScope(scope pcommon.InstrumentationScope) {
-	scope.SetDroppedAttributesCount(0)
+	scope.Attributes().PutStr("pumpkin", "pie")
```

Comment:
🎃🥧

Reply:
I fixed it in the next pr

Reply:
This was by no means a request to fix. I like the 🎃🥧